.github/workflows/test_on_push.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
     strategy:
       matrix:
-        tarantool-version: ["1.10", "2.2", "2.3", "2.4", "2.5"]
+        tarantool-version: ["1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
```

Comment:
Probably we should to add "2.8" as well to test that https://github.com/tarantool/tarantool/commit/e17a53843fa19603213d5a0f4d08750335223436 works ok with https://github.com/tarantool/crud/pull/121